android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -862,7 +862,7 @@ public void run() {
     protected void onDestroy() {
         super.onDestroy();
         unbindService(customTabsServiceConnection);
-        if(disposable != null && !disposable.isDisposed())
+        if(disposable != null)
             disposable.dispose();
```

Comment:
Make a final composite disposable and dispose that

Reply:
yeah sure!

Reply:
and inside we UserProfileActivity should we use composite disposable or using disposable is okay since there is only one Disposable event?

Reply:
Use composite as the subscribe can be triggered many times